### ISSUE TITLE
Current total filtering state is unclear to some users

### ISSUE THREAD

**Issue by zachmullen** [XDATA-Year-3/geoapp#150]
This could be remedied a couple of ways:
- When data is refiltered, we could reset the chart date range to the max

or
- We could show an information panel somewhere that details the current filtered state that is the intersection of the data filters and the chart filters.

**Comment by manthey** [comment]
I don't think we want to reset the chart date range, as that could be annoying when someone is trying to use it as a filter (e.g., does "power out" show better data than "power outage").

**Comment by jamiesnape** [comment]
One user was irate about this. Unfortunately, so much so that they had no constructive feedback on what they actually wanted (or anything after that).
